_includes/tutorials/kafka-connect-datagen-ccloud/kafka/markup/dev/create-topic.adoc

```diff
@@ -0,0 +1,5 @@
+Create a Kafka topic called `mytopic` in Confluent Cloud.
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/create-topic.sh %}</code></pre>
++++++
```

Comment:
FYC - add text showing the result of successful topic creation.

Reply:
Ack, resolved in https://github.com/confluentinc/kafka-tutorials/pull/496/commits/5d3dd56cb80808f3662e3794a54cd93feb7fc212